app/models/answer.rb

```diff
@@ -1,3 +1,20 @@
 class Answer < ApplicationRecord
   belongs_to :question
+  scope :right, -> { where(correct: true) }
+  validates :body, presence: true
+  validate :validate_max_answers, on: :create
+  # validate :validate_minimum_answers, on: :destroying???????
+  #
+  private
+
+  def validate_max_answers
+    if Question.find(question_id).answers.count >= 4
```

Comment:
```suggestion
    if question.answers.count >= 4
```
пользуемся ассоциациями